src/core_modules/capture-core/components/Pages/MainPage/EventWorkingLists/Redux/eventWorkingListsRedux.types.js

```diff
@@ -1,5 +1,92 @@
 // @flow
+import type { EventProgram } from '../../../../../metaData';
+import type {
+    CancelLoadTemplates,
+    CancelLoadView,
+    CancelUpdateList,
+    Categories,
+    ChangePage,
+    ChangeRowsPerPage,
+    ClearFilter,
+    CustomMenuContents,
+    CustomRowMenuContents,
+    FiltersData,
+    LoadedContext,
+    LoadTemplates,
+    LoadView,
+    SelectRestMenuItem,
+    SelectRow,
+    SelectTemplate,
+    SetColumnOrder,
+    Sort,
+    StickyFilters,
+    UnloadingContext,
+    UpdateFilter,
+    UpdateList,
+    UpdateTemplate,
+    WorkingListTemplate,
+    WorkingListTemplates,
+} from '../../WorkingLists';
+import type {
+    CustomColumnOrder,
+    EventsMainProperties,
+    EventsDataElementValues,
+} from '../types';
 
-export type Props = $ReadOnly<{
+export type Props = $ReadOnly<{|
     listId: string,
-}>;
+|}>;
+
+export type EventWorkingListsReduxOutputProps = {|
```

Comment:
I think I made this comment on some later PR but it looks to me that thus type is only used in the `ColumnSetup` module. 

What made you export it from here to then import it in the other module than defining it in the other module directly?